src/runtime/builtin_modules/sys.cpp

```diff
@@ -17,6 +17,7 @@
 #include <cmath>
 #include <langinfo.h>
 #include <sstream>
+#include <gmp.h>
```

Comment:
Do we really need `gmp.h` here?
BTW, can we just hardcode the `sys.long_info` in `test_long`? We can implement is latter. Maybe we will use different approach to implement `long` in future.